views/recipe.pug

```diff
@@ -9,61 +9,66 @@ html(lang='en')
-                                button(id='displayReviewButton') Write A Review
-                            div
-                                button(class='recipe__cookBook--button') Add to Cookbook
-                                //- ask about this for add to cookbooks
+        div(class='header-1')
+            h1=recipe.title
+            h2(class='recipeRating')=recipe.avgRating
+            //- still need to get avgRating
+        div(class='recipe__recipeCard')
+            div(class='recipe__description')
+                each recipeDescription in recipeDescriptions
+                    p=recipeDescription
+                div
+                    img(src=recipe.foodImg, alt=recipe.title)
+                div(class='recipe__ingredients')
+                    h3(class='header-3') Ingredients
+                    ul
+                        each recipeIngredient in recipeIngredients
+                            li=recipeIngredient
+                div(class='recipe__directions')
+                    h3(class='header-3') Directions
+                    ul
+                        each recipeDirection in recipeDirections
+                            li=recipeDirection
+                    div(class='reviewCookbookWrap')
+                        div(class='recipe__reviewButton')
+                            button(id='displayReviewButton') Write A Review
+                        div
+                            button(class='recipe__cookBook--button') Add to Cookbook
+                            //- ask about this for add to cookbooks
 
-                    div(class='recipe__reviews')
-                        h3(class='header-3') Reviews
-                        div(class='recipe__review')
-                            each recipeReview in recipeReviews
-                                div(class='recipe__review--content')
-                                    p(class='recipe__review--username')=recipeReview.User.userName
-                                    span=recipeReview.revContent
-                            //- how to oder the reviews
+                div(class='recipe__reviews')
+                    h3(class='header-3') Reviews
+                    div(class='recipe__review')
+                        each recipeReview in recipeReviews
+                            div(class='recipe__review--content')
+                                div(class='reviewContent__name--andRating')
+                                    span(class='userName_and_rating')= `${recipeReview.User.userName} ${recipeReview.rating}`
+                                    -let n=0
+                                    while n<recipeReview.rating
+                                        span(id='the-stars') &#127775
+                                        -n++
+                                div(class='reviewContent__the--content')
+                                    p(id='review__content')=recipeReview.revContent
+                        //- how to oder the reviews
```

Comment:
If you want to leave this as a TODO, please be sure you are referencing a specific github issue here as well.

Reply:
Removed, we already decided on listing reviews by date posted.